File: sdkwork-ai-sdk-go/types/document_add_request.go
package types

// Document add request
type DocumentAddRequest struct {
	Content string `json:"content"`
	Title string `json:"title"`
	Type string `json:"type"`
	Metadata map[string]interface{} `json:"metadata"`
	ChunkConfig ChunkConfig `json:"chunkConfig"`
	KnowledgeBaseId string `json:"knowledge_base_id"`
}

File: sdkwork-ai-sdk-typescript/src/types/document-add-request.ts
import type { ChunkConfig } from './chunk-config';

/** Document add request */
export interface DocumentAddRequest {
  /** 文档内容 */
  content: string;
  /** 文档标题 */
  title?: string;
  /** 文档类型 */
  type?: string;
  /** 文档元数据 */
  metadata?: Record<string, unknown>;
  /** 切片配置 */
  chunkConfig?: ChunkConfig;
  /** 知识库ID */
  knowledge_base_id: string;
}
